_appmap/importer.py

```diff
@@ -189,6 +189,9 @@ def instrument_functions(filterable, selected_functions=None):
         fm = FilterableMod(mod)
         if cls.filter_chain.filter(fm):
             instrument_functions(fm, package_functions.get(fm.fqname))
+        elif cls.filter_chain.filter(fm):
+            instrument_functions(fm)
```

Comment:
1) The two if statements at lines 190 and 192 seem to be the same.  Does `cls.filter_chain.filter()` cause some side effect that would make it return false on the first call and true in the second?  I added debug statements and didn't see the body in the `elif` part to get executed.

What might be happening is that `MatcherFilter` returns either true or false but `NullFilter` always returns false, necessitating the second call to instrument the function.  If so, maybe add a comment in the code explaining part of this.

2) I still see the two functions getting wrapped.  I assume this due to https://github.com/getappmap/appmap-python/issues/224 which we'll park for now.
```
will wrap fn_name from_yaml
will wrap fn_name to_yaml
```

Reply:
Re 1: good catch. This code was supposed to mirror the test at line 202, but I copied and pasted the wrong code. It's fixed now.

Re 2: yes, that's right.